ci/infra/testrunner/kubectl/kubectl.py

```diff
@@ -16,75 +16,75 @@ def __init__(self, conf, platform):
 
     def create_deployment(self, name, image):
         print("create a new deployment {}".format(name))
+        cmd = "create deployment {name} --image={image}".format(name=name, image=image)
         try:
-            self._run_kubectl("create deployment {name}  --image={image}"
-                                .format(name=name, image=image))
+            self._run_kubectl(cmd)
         except Exception as ex:
-            raise Exception("Error executing cmd {}") from ex
+            raise Exception("Error executing cmd {}: {}".format(cmd, ex)) from ex
```

Comment:
I think we don't need to include the Exception in the message as we are using it in the `from ex`

Reply:
ok, changing it